### PR TITLE
Update PyTorch support to work with v1.3

### PR DESCRIPTION
**Pull request by steven-johnson**
attn @mgharbi

### CODE REVIEW

File: src/runtime/HalidePyTorchHelpers.h [hunk 1]
```diff
@@ -48,6 +48,31 @@ inline void check_type(at::Tensor &tensor) {
     AT_ERROR("Scalar type ", tensor.scalar_type(), " not handled by Halide's PyTorch wrapper");
 }
 
+// TODO: if PyTorch exposes any variable with the API version,
+// I haven't found it in source or documentation; for now, we'll sniff
+// this macro's existence to infer that we are building with v1.3+ (vs 1.2)
+#ifdef AT_FORALL_SCALAR_TYPES_WITH_COMPLEX_AND_QINTS
```
Comment (shoaibkamil) [inline src/runtime/HalidePyTorchHelpers.h]: This is unfortunate but seems unavoidable.

File: src/runtime/HalidePyTorchHelpers.h [hunk 2]
```diff
@@ -59,12 +84,18 @@ AT_FORALL_SCALAR_TYPES_WITH_COMPLEX(HL_PT_DEFINE_TYPECHECK);
 
 #undef HL_PT_DEFINE_TYPECHECK
 
+#endif
```
Comment (shoaibkamil) [inline src/runtime/HalidePyTorchHelpers.h]: Nit: It'd be nice to see what this `#endif` corresponds to.

**Comment by shoaibkamil** [comment]
Minor nit, otherwise LGTM

File: src/runtime/HalidePyTorchHelpers.h (hunk 2 above)
Comment (steven-johnson) [reply src/runtime/HalidePyTorchHelpers.h]: Done

File: src/runtime/HalidePyTorchHelpers.h (hunk 1 above)
Comment (mgharbi) [reply src/runtime/HalidePyTorchHelpers.h]: I don't know of a better solution, either.

**Pull request merged**